### PR TITLE
Remove low frequency trend

### PR DESCRIPTION
**Pull request by mirca**

### CODE REVIEW

**Comment by mirca** [comment]
The PRF tests are passing locally. I'm gonna skip them on Travis until we add support for downloading the PRF calibration files.

**Comment by mirca** [comment]
cc @barentsen

File: pyke/lightcurve.py [hunk 1]
```diff
@@ -25,15 +33,53 @@ def __init__(self, time, flux, flux_err=None, quality=None, centroid_col=None,
...
+            centroid_col = np.append(centroid_col, others[i].centroid_col)
+            centroid_row = np.append(centroid_row, others[i].centroid_row)
+
+        return LightCurve(time=time, flux=flux, flux_err=flux_err,
+                          quality=quality, centroid_col=centroid_col,
+                          centroid_row=centroid_row)
+
+    def flatten(self, window_length=101, polyorder=3, **kwargs):
         """
+        Removes low frequency trend using a Savitzky-Golar filter.
```
Comment (barentsen) [inline pyke/lightcurve.py]: It's spelled "GolaY"?

File: pyke/lightcurve.py [hunk 2]
```diff
@@ -25,15 +33,53 @@ def __init__(self, time, flux, flux_err=None, quality=None, centroid_col=None,
...
+
+        Parameters
+        ----------
+        deg : array-like
+            Degree of the polynomial.
```
Comment (barentsen) [inline pyke/lightcurve.py]: You will want to update the docstring to reflect the longer argument names.

File: pyke/lightcurve.py [hunk 3]
```diff
@@ -25,15 +33,53 @@ def __init__(self, time, flux, flux_err=None, quality=None, centroid_col=None,
...
-            return ArcLengthDetrender().detrend(time=self.time, flux=self.flux,
-                                                flux_err=self.flux_err, **kwargs)
-        else:
-            return FirstDifferenceDetrender().detrend(time=self.time, flux=self.flux,
-                                                      flux_err=self.flux_err, **kwargs)
+        trend = signal.savgol_filter(x=self.flux, window_length=window_length,
+                                     polyorder=polyorder, **kwargs)
+        return (LightCurve(time=self.time, flux=trend),
+                LightCurve(time=self.time, flux=self.flux/trend))
+
```
Comment (barentsen) [inline pyke/lightcurve.py]: Let's change the order to return (flattened_lightcurve, trend)

File: pyke/lightcurve.py [hunk 4]
```diff
@@ -25,15 +33,53 @@ def __init__(self, time, flux, flux_err=None, quality=None, centroid_col=None,
         self.centroid_col = centroid_col
         self.centroid_row = centroid_row
 
-    def detrend(self, method='arclength', **kwargs):
+    def stitch(self, *others):
+        time = self.time
```
Comment (barentsen) [inline pyke/lightcurve.py]: We may think about supporting multiple "stitching methods" with an optional `method=` attribute, but it's beyond the scope of this PR!

**Pull request merged**